runtime/browser/runtime_url_request_context_getter.cc

```diff
@@ -139,7 +159,7 @@ net::URLRequestContext* RuntimeURLRequestContextGetter::GetURLRequestContext() {
             net::DISK_CACHE,
             net::CACHE_BACKEND_DEFAULT,
             cache_path,
-            0,
+            GetDiskCacheSize(),
```

Comment:
I thought it'd be easier to store this value somewhere and pass it to this class. Let's not worry about this part for now then.